src/interfaces/standings.interface.ts

```diff
@@ -0,0 +1,51 @@
+  import {Problem} from './problem.interface'
+  import {Member} from './member.interface'
+  
+
+  export interface Standings {
+    contest: Contest
+    problems: Problem[]
+    rows: Row[]
+  }
+  
+  export interface Contest {
+    id: number
+    name: string
+    type: string
+    phase: string
+    frozen: boolean
+    durationSeconds: number
+    startTimeSeconds: number
+    relativeTimeSeconds: number
+  }
+  
+  
+  export interface Row {
+    party: Party
+    rank: number
+    points: number
+    penalty: number
+    successfulHackCount: number
+    unsuccessfulHackCount: number
+    problemResults: ProblemResult[]
+  }
+  
+  export interface Party {
+    contestId: number
+    members: Member[]
+    participantType: string
+    ghost: boolean
+    startTimeSeconds: number
+    room?: number
+    teamId?: number
+    teamName?: string
+  }
+  
+
+  
+  export interface ProblemResult {
```

Comment:
You're missing the optional `penalty` field. https://codeforces.com/apiHelp/objects#ProblemResult